docs/_user_guide/julia_on_jupyter.md

```diff
@@ -1,12 +1,12 @@
-# How Do I Install Julia on JupyterHub?
+ # How Do I Install Julia on JupyterHub?
```

Comment:
There is an extra space in front of the # here that causes the header to not render correctly. Presumably this line is supposed to be removed entirely?